src/Tools/Source/CompilerGeneratorTools/DeployCompilerGeneratorToolsRuntime/project.json

```diff
@@ -2,8 +2,8 @@
   "dependencies": {
     "Microsoft.NETCore.Portable.Compatibility": "1.0.1-beta-23504",
     "Microsoft.NETCore.Platforms": "1.0.1-beta-23504",
```

Comment:
For Microsoft.NETCore.Platforms in particular it is actually used to pull in the runtime specific packages and so it needs to be updated in order to get the newer shim packages (System.Native) which didn't even exist in the beta-23504 timeframe.